src/openMVG/multiview/translation_averaging_solver_l2_chordal.cpp

```diff
@@ -1,35 +1,5 @@
-//  Copyright (c) 2014, Kyle Wilson
```

Comment:
Note that you remove copyright over some code part that have been released before!
Here you are doing two infringement:
- removing the copyright,
- perform a re-licensing without keeping the original license notice.

Reply:
Oh yes, good catch!
It is a mistake. I used a script to clean all headers and it was not mentioned from the top level licensing files.
So it was a bit hidden in the code, which is maybe also a kind of infringement as it is supposed to be delivered during redistribution including redistribution in binary form.
We should maybe consider installing licenses from the build system.

Reply:
1. OpenMVG does not deliver binary form.
2. You're right I will update the COPYRIGHT.md accordingly.

On the other side I don't think there is a clear `standard process` to install the license files. 
 Do you have any recommendation on this point?

Reply:
Such things typically get installed into `/usr/share/doc/<package>/COPYRIGHT`